### PR TITLE
Allows the pa11y domain to be overridden via a variable.

### PR DESCRIPTION
**Pull request by jhedstrom**
This allows `pa11y` to run in GitLab, within Docker locally, etc.

### CODE REVIEW

File: tools/tests/accessibility/pa11y.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 // All non-protons PL paths
 const { componentPaths } = require('../../pl-paths.js');
```
Comment (illepic) [inline tools/tests/accessibility/pa11y.js]:
```js
const { PL_BASE_DOMAIN } = process.env;
```

File: tools/tests/accessibility/pa11y.js [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
 
-const plRoot = 'http://0.0.0.0:8080/app-pl/pl'; // @TODO: move this to config
+// The PL_BASE_DOMAIN environment variable should be set to override the default.
+let plRootDomain = '0.0.0.0:8080';
+if (process.env.PL_BASE_DOMAIN) {
+  plRootDomain = process.env.PL_BASE_DOMAIN;
+}
```
Comment (illepic) [inline tools/tests/accessibility/pa11y.js]:
```js
const plRootDomain = PL_BASE_DOMAIN || '0.0.0.0:8080';
```

Comment (illepic) [reply tools/tests/accessibility/pa11y.js]: Note: still want to get this into config, per Pattern Lab app.

File: tools/tests/accessibility/pa11y.js (hunk 1 above)
Comment (jhedstrom) [reply tools/tests/accessibility/pa11y.js]: Done!

**Pull request merged**